src/js/index.js

```diff
@@ -0,0 +1,21 @@
+const getSettings = async () => {
+  try {
+    const response = await fetch("/settings/read", {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+    const data = await response.json()
+    return data
+  } catch (error) {
+    console.log(error)
+    return {}
+  }
+}
+
+;(async () => {
```

Comment:
I see you hiding there.

You little semicolon... thinking you can sneak past the linter by being at the start of the line...